terraso_backend/apps/auth/urls.py

```diff
@@ -54,4 +55,5 @@
     ),
     path("tokens", csrf_exempt(RefreshAccessTokenView.as_view()), name="tokens"),
     path("logout", csrf_exempt(LogoutView.as_view()), name="logout"),
+    path("token-exchange", csrf_exempt(TokenExchangeView.as_view()), name="token-exchange"),
```

Comment:
Question: I just want to see if I understood this correctly, so, at the end of the OAuth flow the mobile app gets an id_token from the OAuth provider, then with that token the mobile app will create a request to `token-exchange` to get a terraso token, is that correct?

Reply:
Yep, that's correct.